docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
     volumes:
       - .:/app:z
     environment:
-      - SETTINGS__REDIS_URL=redis://redis:6379
+      - SETTINGS__REDIS_URL=redis:6379
```

Comment:
The prefix is automatically added in https://github.com/RedHatInsights/compliance-backend/blob/master/config/initializers/sidekiq.rb#L4 , so it'll fail to load with it